js/views/modals/purchase/Payment.js

```diff
@@ -42,6 +43,7 @@ export default class extends BaseVw {
     this.paymentAddress = options.paymentAddress;
     this.orderId = options.orderId;
     this.isModerated = options.isModerated;
+    this.page = Backbone.history.getFragment().split(/[//?]/)[1];
```

Comment:
What is the purpose of this? Isn't it alway going to be `store`?

Reply:
It's to tell if the payment was made on the purchase screen or the transactions screen.

It returns "store" or "purchases".

Reply:
Can we just capture the whole url instead of piece mealing it? It seems fragile to me to integrate into a common component that we're only going to grab the third section of the url which may be who know's what in a different context.